lib/kb_staging_exporter/Utils/staging_downloader.py

```diff
@@ -409,12 +418,36 @@ def export_to_staging(self, ctx, params):
         
         if not (set(os.listdir(staging_dir)) >= set(files)):
             raise ValueError('Unexpected error occurred during copying files')
+        return files
 
-        returnVal = dict()
-        returnVal['result_dir'] = result_dir
-
-        if generate_report:
-            report_output = self._generate_export_report(files, obj_name, workspace_name)
-            returnVal.update(report_output)
+# TODOS for JSON export:
+# * implement legacy w/ tests
+# * add UI
 
-        return returnVal
+    def export_json_to_staging(self, ctx, params):
+        for p in ['input_ref', 'destination_dir']:
+            if p not in params:
+                raise ValueError('"{}" parameter is required, but missing'.format(p))
+        format = params.get("format")
```

Comment:
```suggestion
        fmt = params.get("format")
```
shadows str.format built-in but might be ok here since we are not using format().

Reply:
yeah, I've literally never seen that builtin used, I figure it's basically legacy at this point

Reply:
👍